setup.py

```diff
@@ -1,21 +1,50 @@
-import codecs
-
-from os import path
 from setuptools import find_packages, setup
 
+LONG_DESCRIPTION = """
+.. image:: http://pinaxproject.com/pinax-design/patches/blank.svg
+    :target: https://pypi.python.org/pypi/pinax-waitinglist/
+
+=================
+Pinax Waiting List
+=================
+
+.. image:: https://img.shields.io/pypi/v/pinax-waitinglist.svg
+    :target: https://pypi.python.org/pypi/pinax-waitinglist/
+.. image:: https://img.shields.io/badge/license-MIT-blue.svg
+    :target: https://pypi.python.org/pypi/pinax-waitinglist/
+
+.. image:: https://img.shields.io/circleci/project/github/pinax/pinax-waitinglist.svg
+    :target: https://circleci.com/gh/pinax/pinax-waitinglist
+.. image:: https://img.shields.io/codecov/c/github/pinax/pinax-waitinglist.svg
+    :target: https://codecov.io/gh/pinax/pinax-waitinglist
+.. image:: https://img.shields.io/github/contributors/pinax/pinax-waitinglist.svg
+    :target: https://github.com/pinax/pinax-waitinglist/graphs/contributors
+.. image:: https://img.shields.io/github/issues-pr/pinax/pinax-waitinglist.svg
+    :target: https://github.com/pinax/pinax-waitinglist/pulls
+.. image:: https://img.shields.io/github/issues-pr-closed/pinax/pinax-waitinglist.svg
+    :target: https://github.com/pinax/pinax-waitinglist/pulls?q=is%3Apr+is%3Aclosed
+
+.. image:: http://slack.pinaxproject.com/badge.svg
+    :target: http://slack.pinaxproject.com/
 
-def read(*parts):
-    filename = path.join(path.dirname(__file__), *parts)
-    with codecs.open(filename, encoding="utf-8") as fp:
-        return fp.read()
+
+``pinax-waitinglist`` is a waiting list app for Django sites.
+
+
+Supported Django and Python Versions
+------------------------------------
+
+* Django 1.8, 1.10, 1.11, and 2.0
+* Python 2.7, 3.4, 3.5, and 3.6
+"""
 
 
 setup(
     author="Pinax Team",
     author_email="team@pinaxproject.com",
     description="a Django waiting list app",
     name="pinax-waitinglist",
-    long_description=read("README.rst"),
+    long_description=LONG_DESCRIPTION,
     version="1.3.1",
```

Comment:
bump version to 2.0.0, matching guidance and the documentation changelog section